backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandraRule.java

```diff
@@ -145,4 +145,12 @@ public void unpause() {
         client.unpauseContainerCmd(cassandraContainer.getContainerId());
     }
 
+    public TestRule getLifecycleTestRule(int iterationsBetweenRestart) {
```

Comment:
I don't understand why we are hiding the builder here, a builder is supposed to be easy to use, if it's not, we need to change the builder, not workaround it.

Reply:
:+1: